Autopilot/PathManager/Inc/trackFinder.hpp

```diff
@@ -1,7 +1,9 @@
 #pragma once
```

Comment:
I'm confused. I think nixon was working on this and we had agree to have this internal to Sensorfusion rather than have it as a free standing module.

Reply:
yeah it's supposed to be in sensor fusion, it's merged in so that Tony can access this function when he eventually merges. This file shouldn't be used anywhere EXCEPT sensorfusion.

Reply:
Ok, truth be told this should probably just be a static function on the inside of his module rather than a stand alone file, but Tony can fix that when he deals with it.